### DISCUSSION TITLE
Possible OL correlation to Airflow/Celery issue

### DISCUSSION THREAD

**Discussion by seanmuth** [OpenLineage/OpenLineage#1423] [General]
Airflow GH Issue: https://github.com/apache/airflow/issues/28431

In all of the reproduction attempts, I'm able to reproduce this issue, except when I have `OPENLINEAGE_DISABLED=true` set in my env vars.

Unclear what the correlation actually is, but after a week's worth of Stalled Running Tasks being reproduced consistently in other deployments, but not in one with OL disabled (no other changes) there's a strong correlation here.

I'd rather not have two issues going for this, so I decided a discussion here, pointing to the issue on airflow should suffice for now.

**Comment by conorbev** [comment]
I think this could be related: https://github.com/OpenLineage/OpenLineage/pull/1398
